README.md

```diff
@@ -482,14 +488,77 @@ print(stat.size)
 
 ## Promote Docker image
 Promotes a Docker image in a registry to another registry.
-```
+```python
 from artifactory import ArtifactoryPath
 
 path = ArtifactoryPath("http://example.com/artifactory")
 
 path.promote_docker_image("docker-staging", "docker-prod", "my-application", "0.5.1")
 ```
 
+## Builds
+### Get all builds
+Since: 2.2.0  
+Security: Requires a privileged user (can be anonymous).  
+From version 6.6, requires read permission for the build or basic read.
+~~~python
+from artifactory import ArtifactoryBuild
```

Comment:
We should refactor it - let's add actual `ArtifactoryBuild` class in order to be able to work with them in more OOP-style:
```python
artifactory = ArtifactoryBuildManager("https://repo.jfrog.org/artifactory", project="project_name")

builds: List[ArtifactoryBuild] = build_manager.all_builds
build = build[0]

build_runs: List[ArtifactoryBuildRun] = build1.build_runs
build_run1, build_run2 = build_runs[0], build_runs[1]

# Get info - dict
build_run1.info

# Compare them
build_run1.diff(build_run2)


# Promote
build_run1.promote(ci_user="admin", properties={                               
     "components": ["c1","c3","c14"],
     "release-name": ["fb3-ga"]
 }
)
```
So there should be classes:
1. ArtifactoryBuildManager
1. ArtifactoryBuild
1. ArtifactoryBuildRun

Under the hood 2 and 3 will use `ArtifactoryBuildManager` to call API methods

Reply:
agree with proposal, this will require additional effort

will work on it once `repo.jfrog.org/artifactory` will go live back (now is down for some reason)